repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/git/GitHelper.java

```diff
@@ -404,4 +406,27 @@ public void removeNotificationFromTravisYML(File directory, AbstractStep step) {
         }
     }
 
+    public void removeGhOauthFromCreatedFilesToPush(File directory, List<String> fileNames) {
+        String ghOauthPattern = "--ghOauth\\s+[\\w]+";
+        for (String fileName : fileNames) {
+            File file = new File(directory, fileName);
+
+            if (!file.exists()) {
+                getLogger().warn("The file "+file.toPath()+" does not exist.");
+            } else {
+                Charset charset = StandardCharsets.UTF_8;
+                try {
+                    String content = new String(Files.readAllBytes(file.toPath()), charset);
+                    String updatedContent = content.replaceAll(ghOauthPattern, "");
```

Comment:
Maybe it's better to add a tag "[REMOVED]" instead of nothing.

Reply:
Done.